packages/react-components/react-select/src/stories/Select/SelectDefault.stories.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { Select, SelectProps } from '../index';
+import { Select, SelectProps } from '@fluentui/react-select';
 import { useId } from '@fluentui/react-utilities';
```

Comment:
_nit:_ You could split this up into:

```suggestion
import { Select } from '@fluentui/react-select';
import { useId } from '@fluentui/react-utilities';
import type { SelectProps } from '@fluentui/react-select';
```